packages-backend/eslint-config-node/index.js

```diff
@@ -0,0 +1,222 @@
+// This is a workaround for https://github.com/eslint/eslint/issues/3458
+require('@rushstack/eslint-patch/modern-module-resolution');
+
+const { statusCode, allSupportedExtensions } = require('./helpers/eslint');
+// const { craRules } = require('./helpers/rules-presets');
+
+/**
+ * @type {import("eslint").Linter.Config}
+ */
+module.exports = {
+  root: true,
+
+  parser: '@babel/eslint-parser',
+
+  parserOptions: {
+    sourceType: 'module',
+    requireConfigFile: false,
+    /**
+     * @type {import('@babel/core').TransformOptions}
+     */
+    babelOptions: {
+      presets: [['@babel/preset-env', { targets: { node: 'current' } }]],
```

Comment:
Should we maybe allow passing this as an option to not force anybody using this config to use the current version of Node.js? 🤔

Reply:
We can make it configurable yes.

Reply:
Hmm actually I'm not sure how to make it configurable, it's not like Babel.

Btw, the `current` node version is the same as `process.versions.node`. This respects the active node version, like when using `.nvmrc` file.

<img width="512" alt="image" src="https://user-images.githubusercontent.com/1110551/160370838-cd95e767-5997-4e00-8c2b-3d4766e756dc.png">

I think it should be fine like this. Or do you have a more specific use case in mind?

Reply:
Ah okay. I understood differently what `current` means. If it's `process.env.node` it should be fine. As long as people respect the nvmrc.

Reply:
Yes that's what it says in the Babel docs: https://babeljs.io/docs/en/options#targetsnode

<img width="841" alt="image" src="https://user-images.githubusercontent.com/1110551/160378956-130d912c-c1d8-41e1-8885-1c9b4d50ceaf.png">

Reply:
Sounds good. I think in internal services I just wanted to be more careful when it's around building.